ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/PeerDiscoveryAgent.java

```diff
@@ -314,6 +315,11 @@ protected void handleOutgoingPacket(final DiscoveryPeer peer, final Packet packe
                 if (err instanceof SocketException && err.getMessage().contains("unreachable")) {
                   LOG.debug(
                       "Peer {} is unreachable, packet: {}", peer, wrapBuffer(packet.encode()), err);
+                } else if (err instanceof UnsupportedAddressTypeException) {
+                  LOG.warn(
+                      "Sending to peer failed - this is likely due to disabled ipv6 at runtime {}, packet: {}",
```

Comment:
I have still not fully understood when this is happening. Is that because our local machine does not support IPv6, or is it the peer's node that does not support it?
We might want to make that very clear here.

Reply:
This only happens in docker when certain ipv6 supports are turned off out of the box - this has nothing to do with the peer node.